mobile/lib/interfaces/album.interface.dart

```diff
@@ -1,21 +1,43 @@
 import 'package:immich_mobile/entities/album.entity.dart';
 import 'package:immich_mobile/entities/asset.entity.dart';
 import 'package:immich_mobile/entities/user.entity.dart';
+import 'package:immich_mobile/interfaces/database.interface.dart';
 
-abstract interface class IAlbumRepository {
-  Future<int> count({bool? local});
+abstract interface class IAlbumRepository implements IDatabaseRepository {
   Future<Album> create(Album album);
-  Future<Album?> getById(int id);
+
+  Future<Album?> get(int id);
+
   Future<Album?> getByName(
     String name, {
     bool? shared,
     bool? remote,
   });
+
+  Future<List<Album>> getAll({
```

Comment:
In the upcoming layout, I will need to get all albums, both shared and owned. Can it be done with this interface in a single invocation?

Reply:
it can; just do not pass `shared`, `ownerId`. keep both null. it will provide all albums (including local unless you set `remote: true`)